ml-agents/mlagents/trainers/torch/model_serialization.py

```diff
@@ -74,6 +74,13 @@ def __init__(self, policy):
             "is_continuous_control",
             "action_output_shape",
         ]
+        if self.policy.action_spec.continuous_size > 0:
```

Comment:
Following the logic in https://github.com/Unity-Technologies/ml-agents/pull/4655/files#r524491798, we should only include `action` and `action_output_shape` if `continuous_size==0 or discrete_size==0`

Reply:
fixed this.